.github/goodtables_report.py

```diff
@@ -43,8 +46,43 @@
 ERROR_FILE = os.environ['ERROR_FILE']
```

Comment:
On re-review I was confused by this variable name. Is it accurate to call it an error file? Is it not just validation output?